addons/event_crm/models/event_registration.py

```diff
@@ -265,9 +265,9 @@ def _get_lead_description(self, prefix='', line_counter=True, line_suffix=''):
         ]
         description = (prefix if prefix else '') + Markup("<br/>")
         if line_counter:
-            description += Markup("<ol>") + Markup('').join(reg_lines) + Markup("</ol>")
+            description += Markup("<ol>") + Markup(''.join(reg_lines)) + Markup("</ol>")
```

Comment:
The original method should return markup, otherwise we don't know what we're saying is valid actually is.

Reply:
Sorry :zipper_mouth_face:  I didn't see that message from the backport
And I pushed the code

But, yes that fix is good enough :wink: :+1:

Reply:
@nash-odoo No problem

Not sure what we're doing in the end, do you want to cherry-pick it or should we let @nboulif backport his PR and link that on the task instead?

Either works, let's just avoid having 2 people do it ^^'

Reply:
Hello @nboulif,
can you please backport your PR? (https://github.com/odoo/odoo/pull/133736)
cc - @reth-odoo 

Thanks :smile: